db/migrate/20190719185254_add_trigger_to_target_limit_count_by_user.rb

```diff
@@ -7,7 +7,7 @@ def up
         BEGIN
           count_targets :=  (SELECT COUNT(t.*) FROM targets t WHERE NEW.user_id = t.user_id);
 
-          IF count_targets >= #{Api::V1::Target::MAX_TARGETS_PER_USER} THEN
+          IF count_targets >= #{Target::MAX_TARGETS_PER_USER} THEN
```

Comment:
Be careful when updating migrations, this will not run unless you make a rollback and migrate again.